### PR TITLE
first pass tabs and nav

### PR DESCRIPTION
**Pull request by collin**
Hey, just trying out some tabs and navigation.

These will be used for the AVClub navigation. Together the markup will look a little something like this:

```html
<nav>
  <bulbs-nav-toggle nav-name="panel-a">Panel A</bulbs-nav-toggle>
  <bulbs-nav-toggle nav-name="panel-b">Panel B</bulbs-nav-toggle>
</nav>

<bulbs-nav-panel nav-name="panel-a">
  <h1>Panel A</h1>
  <bulbs-tabs>
    <bulbs-tab-strip>
      <bulbs-tab-item tab-name="a">A</bulbs-tab-item>
      <bulbs-tab-item tab-name="b">B</bulbs-tab-item>
    </bulbs-tab-strip>
    <bulbs-tab-content tab-name="a">Tab A</bulbs-tab-content>
    <bulbs-tab-content tab-name="b">Tab B</bulbs-tab-content>
  </bulbs-tabs>
</bulbs-nav-panel>

<bulbs-nav-panel nav-name="panel-b">
  <h1>Panel B</h1>
  <bulbs-tabs>
    <bulbs-tab-strip>
      <bulbs-tab-item tab-name="a">A</bulbs-tab-item>
      <bulbs-tab-item tab-name="b">B</bulbs-tab-item>
    </bulbs-tab-strip>
    <bulbs-tab-content tab-name="a">Tab A</bulbs-tab-content>
    <bulbs-tab-content tab-name="b">Tab B</bulbs-tab-content>
  </bulbs-tabs>
</bulbs-nav-panel>
```

See also: https://github.com/theonion/omni/blob/avc-mobile-nav-collin/avclub/avclub/templates/navigation/nav_panels_modern.html

And in action: http://avc-mobile-nav-collin.test.avclub.com/login/

### CODE REVIEW

**Comment by collin** [comment]
[<bulbs-nav-panel>](https://trello.com/c/diBiFPHR/32-bulbs-nav-panel)

**Comment by collin** [comment]
[<bulbs-nav-toggle>](https://trello.com/c/Wqgzcfvr/31-bulbs-nav-toggle)

**Comment by collin** [comment]
[<bulbs-tab-toggle>](https://trello.com/c/LsojnRte/33-bulbs-tab-toggle)

**Comment by collin** [comment]
[<bulbs-tab-panel>](https://trello.com/c/pk2bVTHv/34-bulbs-tab-panel)

**Comment by MelizzaP** [comment]
@collin what's the status on this branch? Hoping release the nav on Monday

File: elements/bulbs-tabs/bulbs-tabs.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  &:last-of-type {
+    margin-right: auto;
+  }
+}
+
+bulbs-tab-content {
+  display: none;
+
+	&.bulbs-tab-content-active {
+		display: block;
```
Comment (spra85) [inline elements/bulbs-tabs/bulbs-tabs.scss]: Trivial, but can you change these to two spaces to stay consistent?

**Pull request merged**